flang/include/flang/Common/real.h

```diff
@@ -13,6 +13,7 @@
 // The various representations are distinguished by their binary precisions
 // (number of explicit significand bits and any implicit MSB in the fraction).
 
+#include "flang/Runtime/api-attrs.h"
```

Comment:
Should api-attrs.h be moved to Common?

Reply:
Yes, I think `Common` will be more appropriate.  Thanks! I will move it.